app/models.ts

```diff
@@ -4,7 +4,10 @@ export type ID = number;
 export type Dateish = Moment | Date | string;
 export type ActionGrant = Array<string>;
 export type IcalToken = string;
-export type EventTimeType = 'activationTime' | 'startTime';
+export enum EventTime {
+  activate = 'activationTime',
+  start = 'startTime',
```

Comment:
We're not super consistent about it, but we use PascalCase or SCREAMING_SNAKE_CASE for enum values elsewhere. I suggest we descide on one of these and use it everywhere.
TypeScript docs use PascalCase, but I have no strong preference.

Reply:
Yeah. I do agree, but I'm kinda lazy and didn't want to rename all these so I kept the original name. It's just to minimize the amount of changes